.github/workflows/ci-unix-static-av2.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: ctest -j $(getconf _NPROCESSORS_ONLN) --output-on-failure
       - name: Check static link bundling
         run: |
-          cc -o avifenc  -I./apps/shared -I./third_party/iccjpeg -I./include apps/avifenc.c \
+          $CC -o avifenc  -I./apps/shared -I./third_party/iccjpeg -I./include apps/avifenc.c \
```

Comment:
(This comment is just a discussion, not a request for change.)

Note the use of the C compiler `$CC` and the `-lstdc++` linker option. This may not be portable because there may be other C++ runtime code we need to link with to support the C++ code in build/libavif.a. Ideally we should compile the .c file into .o files, and then use the C++ compiler `$CXX` to link all the .o files and build/libavif.a into the avifenc executable.